2014-09-11 09:53:13  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:100960 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-11 14:17:10  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:15938278 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-11 14:17:13  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:15941306 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-11 14:17:35  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:15962949 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
